### ISSUE TITLE
help requested

### ISSUE THREAD

**Issue by sorosj** [john30/ebusd-configuration#44]
I have bougth a Saunier Duval Semiatec Condense 24 type boyler.
I like to use an e-bus ethernet gateway with ebusd to control the heater.
Up till now I could read pump state and switch / off heating, nothing else.
I ask if someone has a config file set for this device and hopefully description to.

**Comment by john30** [comment]
Please post your scan result, info result and grab result, i.e. run "ebusctl scan full", wait a minute, run "ebusctl scan result", "ebusctl info", "ebusctl grab result", and post the output here.

**Comment by sorosj** [comment]
Here is the requested info:

localhost: root@gizi:/etc/ebusd/vaillant# ebusctl scan full
done

root@gizi:/etc/ebusd/vaillant# ebusctl scan result
1 scan(s) still running
08;Vaillant;BAI00;0505;3302;21;16;24;0010020408;2010;005474;N3

root@gizi:/etc/ebusd/vaillant# ebusctl info
version: ebusd 2.1.422b8d6
signal: acquired
symbol rate: 14
masters: 2
messages: 23
address 03: master #11
address 08: slave #11, scanned "MF=Vaillant;ID=BAI00;SW=0505;HW=3302", loaded "vaillant/08.bai.csv"
address 31: master #8, ebusd
address 36: slave #8

root@gizi:/etc/ebusd/vaillant# ebusctl grab result
empty

**Comment by john30** [comment]
I'm afraid, this variant of BAI is not yet supported.
Please let ebusd run for one day or so and then post the result of "ebusctl grab result" again. Some messages should appear there.
Is it correct that you don't have any controller attached to your system?

**Comment by sorosj** [comment]
What is a controller?

**Comment by john30** [comment]
a unit that controls your heater and adapts heat demand depending on e.g. outside temperature.

**Comment by sorosj** [comment]
Yes, there is no controller currently, since I do not understand exactly how to control a boyler over ebusd.
If I can manually control, I will implement a controller.
My final goal is to write a module to my home automation system based on your libebusd.

Last night no grab output collected.

**Comment by john30** [comment]
manually controlling your heater is not an easy task.
since your BAI is not supported yet, this will even be more difficult, because you have no way to verify any message that you willl send to the device...

**Comment by sorosj** [comment]
What do you suggest?

**Comment by sorosj** [comment]
Would you so kind to describe more detailed how to do this?

**Comment by sorosj** [comment]
I have one more question. How can I set lead water temperature and burner state (switch on/off)?

**Comment by john30** [comment]
look at the mcmode.inc/hcmode.inc definitions and try those. but handle with care

**Comment by sorosj** [comment]
In mcmode.inc/hcmode.inc definitions there are only read lines.

**Comment by john30** [comment]
you need to pick the german inc file for that one, the english one is not yet updated

**Comment by sorosj** [comment]
The german file contains the same. No difference.
Now I am trying to send hex commands, but cannot switch burner on or off.
I ask if the standard eBus commands has to be executed by a device or only Vaillant specific ones?
I tried "ebusctl hex  08050105aa60601064" command to start burner and received a 00 as reply, but nothing happened. I have been stuck at this problem.

**Comment by john30** [comment]
the mcmode.inc in german contains a lot of Set messages.

**Comment by sorosj** [comment]
I have copied the current german version to /etc/ebusd but received the following error message.

root@gizi:/etc/ebusd/vaillant# ebusctl read Status
ERR: element not found

**Comment by john30** [comment]
didn't you want to adjust your desired temperature? why are you looking for the status?

**Comment by sorosj** [comment]
in mcmode.inc there is no circuit but in ebusctl write command it is obligatory.

**Comment by john30** [comment]
Ah, you don't have a mixer circuit, that explains it. In this case you can't use the new "set" messages therein. you need to wait for hcmode.inc to be extended accordingly.

**Comment by minscof** [comment]
@sorosj , did you succeed changing temperature or stopping the boiler ? I have a saunier duval and I have the same needs ?

**Comment by rzr** [comment]
I also own a ThemaPlus Condens 25 (0010017387) without the thermostat (well I have only the reciever of  “exacontrol e7r c”), any reported success with it ?

if not where should I start, I guess getting the adapter is the 1st step, right ?

PS: is it possible to convert the offline wiki to github's wiki ? I can help on that

**Comment by estatic** [comment]
> @sorosj , did you succeed changing temperature or stopping the boiler ? I have a saunier duval and I have the same needs ?

Have you any chance to make it work?

I have also Saunier duval boiler and @john30 seems I need your help/advice what I need to do next.
As by logs I could see a few things:
- Boiler has been detected as another model (vaillant/bai.308523.inc)
![image](https://github.com/john30/ebusd-configuration/assets/1957771/953ac986-fc86-47d5-9fda-d9b1cdb41cbf)

- Most of parameters has been detected, however, some of them are blank and no any controls available
- HWC current temp shows -61C, I'm not above north pole, seems a wrong calculation
- ebusd continuesly trying to read scan.05 and fail with timeout. I tried different timeouts for everything, i.e. maximum allowed 60 seconds - didn't help
- I enabled raw log recording
- 

So, what kind of steps I needs to be done next?
